packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -246,12 +226,8 @@ class FinancesApp extends React.Component {
               >
                 <Notifications />
                 <BankSyncStatus />
-
                 <StackedRoutes />
-
                 {/*window.Actual.IS_DEV && <Debugger />*/}
```

Comment:
Should this be re-enabled maybe? (it looks like it needs to have some changes made to work with the new location of the spreadsheet APIs, so maybe not)

Reply:
The `Debugger` component or `URLBar`?

`Debugger` - maybe. But either way not really in scope of this PR.

`URLBar` - don't think we need this. It just shows the active route which is something we can nowadays just see in the URL bar of the browser.

Reply:
I was referring to `Debugger`. Agreed that it’s out of scope.